New version: UniversalDiffEq v1.0.1

JuliaRegistrator opened the pull request:
- Registering package: UniversalDiffEq
- Repository: https://github.com/Jack-H-Buckner/UniversalDiffEq.jl
- Created by: @19fitzna
- Version: v1.0.1
- Commit: bb8446b6b52ec831f06cd9bc8acc1c154ec134b3
- Reviewed by: @19fitzna
- Reference: https://github.com/Jack-H-Buckner/UniversalDiffEq.jl/commit/bb8446b6b52ec831f06cd9bc8acc1c154ec134b3#commitcomment-147219289
- Description: Universal differential equations for ecologists 
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
Solved compiling bug when trying to install from General Registry
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd603ff85d68be6ac17ae9206205111efd5a61a87639f15e41b6651183651896e11044c809f06fe6a9ec99e8f3db72226c3182c71de7166727d9e546afa1705dd5d081a862a1b29a2e42c3cc1a21f8ab597bda1f26f07131e74f41b54a20efab5953b4ca9f891b251cf0523153174184a561dfc00d8b279e7ac075cc30b70e5f3d7646091cb333cecb020fb2491e28f02673caca361110105bdbc7c2f33e9e0682e1658e9453e240174223a7a1aa934d1a2c0b4f265def43eadd285941a3443ee3a -->

jarroyoe commented:
[noblock]
Hi, automerging is complaining that our `[compat]` is not upper-bounded for `Lux.jl`, but our `Project.toml` has this in its `[compat]` section:
`Lux = "0.5 - 1.1.0"`

What are we missing?

JuliaRegistrator committed:
New version: UniversalDiffEq v1.0.1

UUID: 791b002b-939e-4b08-8060-ecf413fcb155
Repo: https://github.com/Jack-H-Buckner/UniversalDiffEq.jl.git
Tree: 2fd13dc7977296288107b3f32bc9373becb8f4aa

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
